content/en/about/case-studies/t-mobile/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "T-Mobile"
+linkTitle: "T-Mobile"
+quote: "I can't stress enough how big of an impact it can be to have generic observability instrumentation that can be immediately adopted by your users."
+author:
+    name: "Joe Searcy"
+    image: "/img/authors/joe-searcy.jpg"
+companyName: "T-Mobile"
+companyURL: "https://www.t-mobile.com/"
+logo: "/logos/t-mobile.svg"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+skip_feedback: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 60
+---
+
+This is a story of struggle, tradeoffs, and triumphs. Istio, as you may know, is a mission-critical piece of software for securing and connecting microservices across platforms. However, it can be daunting to introduce, operationalize, or adopt it successfully.
+
+In [this talk from IstioCon 2021](https://events.istio.io/istiocon-2021/sessions/adopting-istio-across-100-clusters-at-t-mobile/), Joe Searcy digs into T-Mobile’s journey of adopting Istio across 100+ clusters to support microservices for fraud detection, billing, sales and APIs across many teams.
```

Comment:
Noting the last command before and/or is inconsistent here. This doc seems to use them, and I suspect it's missing after `sales` if that is the case.